src/command/mod.rs

```diff
@@ -150,7 +150,10 @@ impl ArtCommand {
     /// Convert an a byte buffer to a command.
     pub fn from_buffer(buffer: &[u8]) -> Result<ArtCommand> {
         if buffer.len() < 13 {
-            return Err(Error::MessageTooShort(buffer.to_vec()));
+            return Err(Error::MessageSizeInvalid {
+                message: buffer.to_vec(),
+                allowed_size: 13..1024, // I'm not sure what the actually allowed size is of an artnet packet, if this is wrong feel free to correct it
```

Comment:
See [Art-Net Spec](https://artisticlicence.com/WebSiteMaster/User%20Guides/art-net.pdf) page 8. There is no maximum packet length and a receiver should not check maximum packet length, only that there are at least as many bytes as expected for the corresponding OpCode and further packet fields. 

Take an ArtDmx packet as example. If you send more Data than you specify in your Length field, the parser should only parse the bytes as far as you define in your length field. All extra bytes should be thrown away. This is also the way the Wireshark Art-Net protocol works. 

The shortest packets are ArtSync and ArtPoll at 14 bytes.

Reply:
Fixed